packages/extension/src/background/transactions/transactionMessaging.ts

```diff
@@ -58,9 +58,14 @@ export const handleTransactionMessage: HandleMessage<
             suggestedMaxFee: number.toHex(suggestedMaxFee),
           },
         })
-      } catch (error) {
+      } catch (error: any) {
         console.error(error)
-        return sendToTabAndUi({ type: "ESTIMATE_TRANSACTION_FEE_REJ" })
+        return sendToTabAndUi({
+          type: "ESTIMATE_TRANSACTION_FEE_REJ",
+          data: {
+            error: error.message,
```

Comment:
as we expect a string, lets make sure it is one in any case
```suggestion
            error: error?.message?.toString.?() ?? error?.toString.?() ?? "Unkown error",
```